Docs/Request_Overrides.md

```diff
@@ -181,3 +181,67 @@ Example:
      ...
 }
   ```
```

Comment:
The heading is not formatted correctly

![image](https://github.com/user-attachments/assets/ab4d019e-4bef-4c14-8e93-1d8da503c146)

Reply:
addressed the comment